index.js

```diff
@@ -67,6 +67,45 @@ function toggleColor(element) {
     }
 }
 
+header.addEventListener('click', function(){
+    console.log('title clicked')
+
+    toggleColor(header)
+
+} )
+
 /***** Deliverable 2 *****/
 
+const likeButton = document.querySelector('button.like-button')
+
+likeButton.addEventListener('click',function(){
+    likes.textcontent = `${traveler.likes++} Likes`
+})
+
 /***** Deliverable 3 *****/
+
+const newSightingForm = document.querySelector('form#new-animal-sighting-form')
+
+newSightingForm.addEventListener('submit', function (event) {
+    event.preventDefault() 
+
+    const species = event.target[0].value
+    const video = event.target[1].value
+    const photo = event.target[2].value
+    const description = event.target[3].value
+
+    const lastIndex = traveler.animalSightings.length - 1
+    const newSightingObject = {
+        id: traveler.animalSightings[lastIndex].id + 1,
+        travelerId: 1,
+        species: species,
+        photo: photo,
+        link: video,
+        description: description
+    }
+
+    renderAnimalSightingPost(newSightingObject)
+
```

Comment:
Here, we could include:
`traveler.animalSightings.push(newSightingObject)`
It is a good idea to include this line because of the way we're creating custom IDs. If we want future animal sightings to have a unique ID, then we have to push all new animal sighting objects into the array.